appveyor.yml

```diff
@@ -26,8 +35,8 @@ before_build:
         } Else {
             $env:SHARED="no-shared no-makedepend"
         }
-    - ps: $env:VSCOMNTOOLS=(Get-Content ("env:VS" + "$env:VSVER" + "0COMNTOOLS"))
-    - call "%VSCOMNTOOLS%\..\..\VC\vcvarsall.bat" %VCVARS_PLATFORM%
+    - @echo VCINSTALL_PATH is "%VCINSTALL_PATH%"
```

Comment:
Appveyor doesn't seem to like the `@` there, so remove it, even if that's damn annoying

Reply:
forget to repush it , sorry ;)

Reply:
Repushed now